packages/client/components/DiscussionThread.tsx

```diff
@@ -99,6 +101,7 @@ const DiscussionThread = (props: Props) => {
   const getMaxSortOrder = () => {
     return Math.max(0, ...threadables.map((threadable) => threadable.threadSortOrder || 0))
   }
+  const isCreatingPoll = threadables.some(isLocalPoll)
```

Comment:
We can't use `isViewerTyping` as we do with tasks as polls have too many inputs and other focusable elements so it causes a lot of edge cases.